### DISCUSSION TITLE
Version 1.7.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1124] [Changelog]
## Bugfixes
- #1006 cloud-games do not link to users. (prio-4)
- #1117 project-notes-tab does not work in Firefox. (prio-2)
- #1117 Instructions do not fill empty space in Firefox when using project-notes-tab. (prio-4)
- #1117 project-notes-tab errors when a section is missing. (prio-2)
- #1117 project-notes-tab can call Instructions tab "Notes and Credits". (prio-3)
- #1117 Buttons added by project-notes-tab overlaps with remix credits. (prio-4)
- #1117 project-notes-tab loads on onload, rather than DOMContentLoaded. (prio-4)
- #1119 Colored context menus do not apply when right-clicking the edge. (prio-4)
- #1119 Colored context menus do not apply to procedure arguments. (prio-4)
- #1120 editor-devtools may perform slow operations. (prio-3)
- #1120 Actions performed by editor-devtools required multiple undos when reverting. (prio-4)
- #1120 CSS in editor-devtools may cause encoding error. (prio-3)
- #1125 Update notification uses wrong font. (prio-4)
- #1125 Clicking outside "More Settings" modal does not close the modal. (prio-4)
- #1128 XSS filter does not escape quotes. (prio-4)
- #1139 block-switching does not work after switching languages. (prio-4)
- #1139 block-switching mishandles forever block. (prio-4)
- #1142 hex-color-picker has wrong background color on dark mode. (prio-3)
- #1145 Popup has two scrollbars. (prio-4)
- #1159 Extension option shows incorrect letters when localized. (prio-4)
- #1160 dark-www does not make studio thumbnail border dark. (prio-4)
- #1160 dark-www is not compatible with other addons. (prio-4)
- #1160 dark-www does not make post preview dark. (prio-3)
- #1160 dark-www does not affect syntax highlighting in forums. (prio-4)
- #1164 Uninstall URL still opens in prerelease. (prio-4)
- #1166 confirm-actions runs on studios and topics. (prio-4)
- #1172 scratch-notifier crashes when receiving crafted message. (prio-2)
- #1172 scratch-notifier converts tags in project titles to emojis. (prio-4)
- #1172 scratch-messaging double escapes tags. (prio-4)
- #1187 Popups are misaligned. (prio-4)
- #1189 scratchr2 changes the color of usernames in Messages page. (prio-4)
- #1190 project-notes-tab description has a typo. (prio-4)
- #1196 editor-devtools can remove extension blocks that are not orphaned. (prio-2)
- #1243 Icons are blurry. (prio-4)
- #1245 getMsgCount errors when used in userscripts. (prio-2)
- #1252 Tooltips are misaligned. (prio-4)
- #1258 Site access screen opens on Firefox. (prio-4)
- #1259 dark-www and scratchr2 do not run on some pages. (prio-4)
- #1312 better-featured-project adds scrollbar to wrong places. (prio-4)

## Features
- #507 New addon: semicolon
- #639 New addon: cat-blocks
- #1006 Add a way to disable popups
- #1006 New popup for changing settings
- #1096 Add a way to change Scratch colors (prio-4)
- #1097 New addon: data-category-tweaks-v2 (prio-3)
- #1121 Add block counter in project-info. (prio-4)
- #1127 New addon: better-quoter (prio-4)
- #1131 Add a way to set FPS to values other than 60 (prio-4)
- #1136 Add a way to export and import settings. (prio-3)
- #1177 Support for easter egg addons
- #1217 New addon: mute-project
- #1242 Messaging related addons no longer go to the top

**Comment by cobaltt7** [comment]
Is there an expected date of release for 1.7.0?

**Reply by mxmou** [reply]
[Yes.](https://github.com/ScratchAddons/ScratchAddons/issues/1132#issuecomment-752283682)

**Reply by Marc92020** [reply]
OK I was also wondering
